sdk/src/api/engine/remote/ws/native.rs

```diff
@@ -243,7 +237,7 @@ async fn router_handle_route(
 			return HandleResult::Ok;
 		};
 		trace!("Request {:?}", request);
-		let payload = serialize(&request, endpoint.supports_revision).unwrap();
+		let payload = serialize(&request, true).unwrap();
```

Comment:
Shouldn't we handle the unwrap?

Reply:
yes, but its not trivial. It was an unwrap before, I think it would be better to do that in a different pr